readme.md

```diff
@@ -105,6 +105,7 @@ Follows some playful (just open-source! :warning:) challenges of jobs.
 | pt   | :brazil: [Woop Sicredi](https://www.woopsicredi.com/)               | Angular, React, Stencil                       | [Code →](https://github.com/WoopSicredi/jobs/issues/6)                                  | -                                                                               |
 | en   | :us: [Yodlr](http://getyodlr.com/)                                  | Framework JS, Preprocessor CSS & Task Runners | [Code →](https://github.com/yodlr/frontend-code-challenge)                              | -                                                                               |
 | en   | :us: [ZEPL](http://www.zepl.com/)                                   | Framework JS, Preprocessor CSS & Task Runners | [Code →](https://github.com/ZEPL/front-end-challenge)                                   | -                                                                               |
+| en   | :us: [Jobsity](https://www.jobsity.com/)                            | React, Redux, Jest | [Code →](https://git.jobsity.com/jobsity/ReactChallenge/-/wikis/React-Challenge-Instructions)                                   | -                                       |
```

Comment:
@gabrielmlinassi Could you add it in alphabetical order? Thanks

Reply:
@felipefialho feito